src/CommonDBTM.php

```diff
@@ -1422,6 +1421,7 @@ public function add(array $input, $options = [], $history = true)
                         //Check if we have to automatically fill dates
                         Infocom::manageDateOnStatusChange($this);
                     }
+                    Webhook::raise('new', $this);
                     Plugin::doHook(Hooks::ITEM_ADD, $this);
```

Comment:
Why before the hook and not after?
```suggestion
                    Plugin::doHook(Hooks::ITEM_ADD, $this);
                    Webhook::raise('new', $this);
```